backend/contributions/serializers.py

```diff
@@ -237,6 +249,18 @@ def get_contribution(self, obj):
             return ContributionSerializer(obj.converted_contribution, context=contrib_context).data
         return None
 
+    def get_mission_details(self, obj):
```

Comment:
why is this different from the previous one? I would implement it as a serializer, not as a method that returns a dict, it's not the pythonic/drf way .

Also keep in mind lightserializers and how this would interact